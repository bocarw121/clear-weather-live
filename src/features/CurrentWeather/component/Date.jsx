import React from "react";
import { StyleSheet, Text } from "react-native";

const Date = ({ time }) => {
  return <Text style={styles.text}>Updated: {time}</Text>;
};

export default Date;

const styles = StyleSheet.create({
  text: {
    marginTop: 10,
    marginLeft: 10,
    fontSize: 16,
  },
});
