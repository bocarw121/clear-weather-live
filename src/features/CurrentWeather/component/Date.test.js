import React from "react";

import { render } from "@testing-library/react-native";

import Date from "./Date";

it("Should render the Date", () => {
  const screen = render(<Date />);

  expect(screen).toMatchSnapshot();
});
