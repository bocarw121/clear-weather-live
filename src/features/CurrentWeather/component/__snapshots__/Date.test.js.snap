// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the Date 1`] = `
<Text
  style={
    Object {
      "fontSize": 16,
      "marginLeft": 10,
      "marginTop": 10,
    }
  }
>
  Updated: 
</Text>
`;
